### PR TITLE
Select Move UI Shows PP

### PR DESCRIPTION
**Pull request by thicco-catto**
When selecting a move through the party UI (like using an ether) the current PP and the max PP for each move shows up.
image

### CODE REVIEW

**Review by Flashfyre — CHANGES_REQUESTED** [review]
Is it possible to make this show only for ether/elixir? I really don't want this showing for other situations.

File: src/modifier/modifier-type.ts [hunk 1]
```diff
@@ -1337,7 +1337,7 @@ export function getPlayerShopModifierTypeOptionsForWave(waveIndex: integer, base
 
   const options = [
     [
-      new ModifierTypeOption(modifierTypes.POTION(), 0, baseCost * 0.2),
+      new ModifierTypeOption(modifierTypes.PP_MAX(), 0, baseCost * 0.2),
```
Comment (Flashfyre) [inline src/modifier/modifier-type.ts:1340]: Careful of leaving test changes like this in

**Review by Flashfyre — CHANGES_REQUESTED** [review]

File: src/modifier/modifier-type.ts (hunk 1 above)
Comment (thicco-catto) [reply src/modifier/modifier-type.ts:1340]: Oh yea sorry

File: src/phases.ts [hunk 2]
```diff
@@ -4340,6 +4340,8 @@ export class SelectModifierPhase extends BattlePhase {
           const isMoveModifier = modifierType instanceof PokemonMoveModifierType;
           const isTmModifier = modifierType instanceof TmModifierType;
           const isRememberMoveModifier = modifierType instanceof RememberMoveModifierType;
+          const isPpRestoreModifier = modifierType instanceof PokemonPpRestoreModifierType;
+            //|| modifierType instanceof PokemonPpUpModifierType;
```
Comment (Flashfyre) [inline src/phases.ts:4344]:
```suggestion
```

File: src/ui/party-ui-handler.ts [hunk 3]
```diff
@@ -185,6 +186,7 @@ export default class PartyUiHandler extends MessageUiHandler {
       ? args[4] as PokemonMoveSelectFilter
       : PartyUiHandler.FilterAllMoves;
     this.tmMoveId = args.length > 5 && args[5] ? args[5] : Moves.NONE;
+    this.showMovePp = args.length > 6 && args[6] ? args[6] : false;
```
Comment (Flashfyre) [inline src/ui/party-ui-handler.ts:189]:
```suggestion
    this.showMovePp = args.length > 6 && args[6];
```

**Review by Flashfyre — CHANGES_REQUESTED** [review]

**Review by Flashfyre — APPROVED** [review]

**Pull request merged**